index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
```

Comment:
It would be much better to wrap all of your code inside the main tag. Let's create a main tag just after the body and put all of your code inside it. It is because the body tag should not be served as the wrapper because it can create some problems in stylesheets.